### DISCUSSION TITLE
Intercepting emitted deltas

### DISCUSSION THREAD

**Discussion by CemYil03** [sibiraj-s/ngx-editor#426] [Q&A]
I would be interested in an event emitter that only returns what has changed and where for every input.
I know the yjs-plugin can handle collaboration, but for my scenario it would be really helpful to have full control myself.

I don't know if that is already possible, if not, this can be seen as a feature suggestion.

**Comment by sibiraj-s** [comment]
You might be looking for [prosemirror-changeset](https://github.com/ProseMirror/prosemirror-changeset)?

[a comment by an automated account is not shown]
